addons/website_sale/static/src/js/website_sale.js

```diff
@@ -172,7 +180,18 @@ export const WebsiteSale = publicWidget.Widget.extend(VariantMixin, cartHandlerM
             this._changeAttribute(['.css_attribute_color', '.o_variant_pills']);
         }
     },
-
+    _restoreScrollPosition: function (lastScrollPosition) {
+        const contentEl = document.getElementById('wrapwrap');
```

Comment:
Why use this element specially?
Isn't `document.body` will do the job? :thinking:

Reply:
In our scenario, using `document.body` won't work because the body element has its scrolling disabled with the "overflow" property set to "hidden".
We can also confirm this by checking if the body is scrollable with this code:
```
scroll = document.body.scrollWidth > document.body.clientWidth; 
console.log(scroll);
```
If it returns `false`, it means the body isn't scrollable. That's why I opted for the next available element, which is the div with the id "wrapwrap", because it is scrollable and working as intended.